public/app/plugins/datasource/prometheus/configuration/AzureCredentialsConfig.ts

```diff
@@ -144,13 +56,18 @@ export function setDefaultCredentials(options: DataSourceSettings<any, any>): Pa
   };
 }
 
-export function resetCredentials(options: DataSourceSettings<any, any>): Partial<DataSourceSettings<any, any>> {
+export function resetCredentials(options: AzurePromDataSourceSettings): Partial<AzurePromDataSourceSettings> {
   return {
     jsonData: {
       ...options.jsonData,
-      azureAuth: undefined,
```

Comment:
I think this may still be required for legacy cases

Reply:
@aangelisc I couldn't find any references to this field anymore. According to this [PR](https://github.com/grafana/grafana/pull/39209) from 2021, it seems like it was from when azure auth was first onboarded and enabling it was controlled by a UI toggle that's gone now. It should be safe to remove now.